src/scenes/home/home.js

```diff
@@ -226,7 +231,8 @@ class Home extends Component {
             <AuthenticatedRoute
               exact path="/profile"
               isLoggedIn={CookieHelpers.getUserStatus().signedIn}
-              component={() => (<Profile {...authProps} />)}
+              component={Profile}
```

Comment:
What's the reasoning behind this change? Did you verify id.me verification was still working?